test/ipcprovider.js

```diff
@@ -33,13 +32,13 @@ describe('lib/web3/ipcprovider', function () {
 
     describe('isConnected', function () {
         it('should return a boolean', function () {
-            var provider = new IpcProvider();
+            var provider = new IpcProvider('', net);
```

Comment:
It should also with if you ant providing a net package, as it should load the nom default one